pkg/policy/eventstate.go

```diff
@@ -0,0 +1,196 @@
+package policy
+
+import (
+	"github.com/aquasecurity/tracee/pkg/capabilities"
+	"github.com/aquasecurity/tracee/pkg/config"
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/pkg/pcaps"
+	"github.com/aquasecurity/tracee/pkg/proctree"
+	"github.com/aquasecurity/tracee/pkg/utils"
+)
+
+// TODO: Policies need to hold its own config (per snapshot), and not rely on the global configuration.
```

Comment:
I realized that I was forced to inject `config.Config` in not usual places. So next step is to move config related to eventstates into Policies.